### PR TITLE
fix #3: always send beforeunload and unload events

### LINKED ISSUE #3: One telemetry ping for every page
Rather than only on the pages that have trackers present.
The pings with no trackers can just be empty

### PR DESCRIPTION
**Pull request by groovecoder**

### CODE REVIEW

**Comment by ericawright** [comment]
I'm not getting the `onSecurityChange` event firing from the framescript. This event fires from navigation events and from script loads. On a page with no trackers It fires before the `domContentLoaded`, so it never records that it is a reload. We could send the `sendAsyncMessage("reload"...)` from `domContentLoaded` instead.

File: src/privileged/trackers/framescript.js [hunk 1]
```diff
@@ -74,9 +57,6 @@ addEventListener("DOMContentLoaded", function(e) {
   // and use the "unload" event (or a tab close) for submitting the data.
   content.window.addEventListener("beforeunload", () => {
     // Don't bother if we have no trackers.
```
Comment (ericawright) [inline src/privileged/trackers/framescript.js]: nit, remove the comment

File: src/privileged/trackers/schema.json [hunk 2]
```diff
@@ -38,7 +38,7 @@
         ]
       },
       {
-        "name": "onReloadWithTrackers",
+        "name": "onReload",
         "type": "function",
         "description": "This is called when we find a page that was reloaded and has at least one tracker.",
```
Comment (ericawright) [inline src/privileged/trackers/schema.json]: nit, change the description

File: src/feature.js [hunk 3]
```diff
@@ -44,7 +44,7 @@ class Feature {
 
     // This is a notification to let us know that we should prompt
     // the user whether the page is broken (the user reloaded a tracking page.)
```
Comment (ericawright) [inline src/feature.js]: nit update the comment

**Comment by groovecoder** [comment]
Updated.

**Pull request merged**